test/e2e/app-dir/next-after-app/index.test.ts

```diff
@@ -298,7 +298,7 @@ describe.each(runtimes)('unstable_after() in %s runtime', (runtimeValue) => {
       )
     })
     afterAll(async () => {
-      await sanboxed.cleanup()
+      await sanboxed[Symbol.asyncDispose]()
```

Comment:
this is a little leaky - I wonder if we should keep the cleanup method for cases where we're not able to leverage `using`?

Reply:
This is the only case where we don't need to use `using`. Not ideal, but adding back `clean` would make it more confusing when people use `clean` at the same time of `using`. I can follow up with another PR to restructure this specific test case, right now it's just a workaround.

Reply:
My related concern is that the magicalness of this means that we're hiding the actual underlying cleanup process from someone implementing a sandboxed test. Do we need to enforce some sort of lint rule that ensures `createSandbox` is always invoked in a `using` context?

Reply:
Done - added a runtime guard to ensure sandbox is created via `using`